toolz/itertoolz.py

```diff
@@ -877,3 +877,21 @@ def topk(k, seq, key=None):
     if key and not callable(key):
         key = getter(key)
     return tuple(heapq.nlargest(k, seq, key=key))
+
+
+def peek(iterable):
+    """ Retrieve the next element of a sequence, and an iterable equivalent
+    to the original sequence.
```

Comment:
Docstring should be a single line for the title, followed by an empty line, followed by more text if necessary.